simple-cipher/simple-cipher.js

```diff
@@ -0,0 +1,43 @@
+function Cipher (key) {
+    this.alphabet = "abcdefghijklmnopqrstuvwxyz".split('');
+    if (key && /^[a-z]+$/.test(key)) {
```

Comment:
This whole logic around validation of the key and generating random key should be in separate functions. Right now it unnecessarly makes constructor harder to use and there are unnamed part of logic (line 8-11)